src/coins/liq.rs

```diff
@@ -0,0 +1,254 @@
+use std::collections::HashMap;
+use std::str::FromStr;
+
+use bitcoin::consensus::encode::{deserialize, serialize};
+use bitcoin::util::bip32;
+use bitcoincore_rpc::{Client as RpcClient, RpcApi};
+use elements::{Address, Transaction};
+use liquid_rpc::{bitcoin_hashes::hex::FromHex, json as rpcjson, json::AssetId, LiquidRpcApi};
+use serde_json::Value;
+
+use crate::errors::{Error, OptionExt};
+use crate::network::ElementsNetwork;
+use crate::throw;
+use crate::util;
+use crate::wallet::AddressMeta;
+
+lazy_static! {
+    pub static ref LBTC: rpcjson::AssetId = rpcjson::AssetId::from_hex(
+        "6f0279e9ed041c3d710a9f57d0c02928416460c4b722ae3457a11eec381c526d"
```

Comment:
can't LBTC and EBTC be a &str so they can both be used for `asset()` and `asset_hex()`

Reply:
True, optimized this.